### ISSUE TITLE
Release also for Firefox

### ISSUE THREAD

**Issue by Mte90** [parthraghav/Know-Your-Advertisers#1]
It is possible to release it also in addons.mozilla.org?
It is the same zip file of chrome (I am an ext dev too).

**Comment by ranagwho** [comment]
Hi, @Mte90:
The current release wouldn't work on Mozilla Firefox. There are some files that are chromium specific. Firefox support is one of the future milestones. If you wish to help in the porting work, feel free to create a pull request.

**Comment by Absolucy** [comment]
@parthraghav Out of curiosity, which parts are Chromium-specific? As they both use the WebExtension API.

**Comment by ranagwho** [comment]
@aspenluxxxy You're right! I currently use chrome.runtime API to communicate between extension scripts. Using WebExtension Polyfill would make the extension compatible with Firefox
https://github.com/parthraghav/Know-Your-Advertisers/blob/2b3c405e31bcc047041599e6557e1f0862035187/src/messenger.ts#L15
